src/test/java/com/google/api/codegen/testdata/ruby_main_library.baseline

```diff
@@ -1133,7 +1137,6 @@ module Library
       # @param options [Google::Gax::CallOptions]
       #   Overrides the default settings for this call, e.g, timeout,
       #   retries, etc.
-      # @return [Google::Tagger::V1::AddTagResponse]
```

Comment:
I think we should not just handle 'google.protobuf.empty', in the case here 'AddTagResponse' is also defined as an empty message here, so we should return nil too. When later, if AddTagResponse becomes non-empty, since we need to re-generate the gapic, then the AddTagResponse will be added back as return type.